app/models/profile.rb

```diff
@@ -22,6 +22,7 @@ class Profile < ApplicationRecord
   validates :first_name, :length => { :maximum => 32 }
   validates :last_name, :length => { :maximum => 32 }
   validates :location, :length => { :maximum =>255 }
+  validates :gender, :length => { :maximum =>255 }
```

Comment:
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
- Layout/SpaceInsideHashLiteralBraces: Space inside { detected.
- Layout/SpaceInsideHashLiteralBraces: Space inside } detected.